### ISSUE TITLE
Merge Conflicts

### ISSUE THREAD

**Issue by brianedmunds** [ga-wdi-boston/full-stack-project#788]
After pushing out my most recent commit, my app broke.  It ends up looking like this and is completely unresponsive:

![image](https://cloud.githubusercontent.com/assets/26188276/25597854/3b5e9aa6-2e9f-11e7-9e45-6da55baadf59.png)

I wasn't expecting the changes that I made to cause any harm - maybe I deleted or modified something that I didn't intend to.  In an attempt to fix this problem, I checked out my previous working commit, checked out a new branch, downloaded a zip file of the latest commit that caused the breakage, manually compared the files in the two commits, and copied/pasted the code from my latest commit into the files from my previous working commit line by line to try to isolate the culprit.  The result is that this fixes the problem locally - my app is working as expected in the local branch that I checked out, which now includes the latest changes that I had made.  However, when I attempt to merge that branch into the master branch, the merge fails due to conflicts.  I get the following message in my terminal:

$ git merge revisedfeature
Auto-merging index.html
CONFLICT (content): Merge conflict in index.html
Auto-merging assets/scripts/workouts/events.js
CONFLICT (content): Merge conflict in assets/scripts/workouts/events.js
Automatic merge failed; fix conflicts and then commit the result.

I've made a couple of attempts at resolving those conflicts manually in atom, but I appear to have failed each time as the end result is that my app continues to be broken and displays as it does in the attached screenshot.

Could I please sit with someone for a few minutes to show them what I am seeing, and get some guidance on how to clear up these conflict issues?

**Comment by brianedmunds** [comment]
Also want to add that the following four files were generated in my new branch as part of my efforts to resolve the conflicts.  Not sure what to do with these:

![image](https://cloud.githubusercontent.com/assets/26188276/25598286/187d10e6-2ea2-11e7-824c-b7be2cbc7c4a.png)

**Comment by brianedmunds** [comment]
I found a solution that will supposedly overwrite the master branch with another branch.  It uses something called the "ours" strategy which, according to the documentation that I was able to find, does the following:

"This resolves any number of heads, but the resulting tree of the merge is always that of the current branch head, effectively ignoring all changes from all other branches. It is meant to be used to supersede old development history of side branches. Note that this is different from the -Xours option to the recursive merge strategy."

Given that I have a branch (revisedfeature) that is functioning as expected locally and contains all of the code that I haven been trying to merge into master, this seems like it could be an option that would circumvent the merge conflicts.  The commands would be as follows:

git checkout revisedfeature
git merge -s ours master
git checkout master
git merge revisedfeature

Would this be a safe route to take?  Don't want to pull the trigger on it if there could be some unintended consequences, or if there is a better solution available.

**Comment by brianedmunds** [comment]
Never received any acknowledgement from an instructor about this issue.  After doing some additional reading on the approach that I described above, I felt comfortable enough to give it a try and it resolved my problem.  Closing this out.

**Comment by jordanallain** [comment]
hey brian i was just about to ask you if you wanted to sit down about this. do you want to go over it or did you feel like it was the right solution?

**Comment by brianedmunds** [comment]
Hey Jordan.  It looks like that resolved my problem, and I'm not seeing any negative side effects so far.  I'll keep an eye on it and revisit this if needed.  Thanks for checking in with me.

**Comment by BenGitsCode** [comment]
@brianedmunds Please close the issue for now, with some details on the resolution. You can always reopen it if the problem returns.

**Comment by brianedmunds** [comment]
Hey Ben.  I did close it, and referenced the steps that I described above as having been the resolution.    In summary, I used the following commands, which leverage the "ours" merge strategy:

git checkout revisedfeature
git merge -s ours master
git checkout master
git merge revisedfeature

End result is that my master branch essentially became a mirror image of what I had in my revisedfeature branch, and I didn't have to manually merge anything.  After pushing to my remote repository and deploying, all looks good.
